rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -1179,6 +1179,8 @@ std::vector<std::pair<u32, u32>> spu_thread::dump_callstack_list() const
 
 	bool first = true;
 
+	const v128 gpr0 = gpr[0];
```

Comment:
why are these copies and not references?

Reply:
They must be copies, this code can execute while the the thread is executing cide in the background and still rewriting registers.